test/web-api.test.ts

```diff
@@ -678,6 +679,70 @@ describe('WebApi', () => {
     });
   });
 
+  describe('business_verification', () => {
+    it('successfully sends a business verification job', async () => {
+      const instance = new WebApi('001', 'https://a_callback.cb', Buffer.from(pair.public).toString('base64'), 0);
+      const partner_params = {
+        user_id: '1',
+        job_id: '1',
+        job_type: JOB_TYPE.BUSINESS_VERIFICATION,
+      };
+      const id_info = {
+        country: 'NG',
+        id_type: 'BUSINESS_REGISTRATION',
+        id_number: 'A000000',
+        business_type: 'co',
+      };
+
+      nock('https://testapi.smileidentity.com')
+        .post('/v1/business_verification', (body) => {
+          expect(body.partner_id).toEqual('001');
+          expect(body.country).toEqual(id_info.country);
+          expect(body.id_type).toEqual(id_info.id_type);
+          expect(body.id_number).toEqual(id_info.id_number);
+          expect(body.business_type).toEqual(id_info.business_type);
+          expect(body.partner_params.job_type).toEqual(JOB_TYPE.BUSINESS_VERIFICATION);
```

Comment:
Prefer to use one large expect here where you can verify that you're only sending the keys you expect to send. 
```suggestion
          expect(body.partner_id).toEqual('001');
          expect(body.country).toEqual(id_info.country);
          expect(body.id_type).toEqual(id_info.id_type);
          expect(body.id_number).toEqual(id_info.id_number);
          expect(body.business_type).toEqual(id_info.business_type);
          expect(body.partner_params.job_type).toEqual(JOB_TYPE.BUSINESS_VERIFICATION);
```

Also consider moving this out of the post using a jest.fn so you can validate the expectation below.